unbxdSearch.js

```diff
@@ -440,8 +502,58 @@ var unbxdSearchInit = function(jQuery, Handlebars) {
             return q;
         },
         getFacetStats: "",
-        processFacetStats: function(obj) {},
-        setDefaultFilters: function() {},
+        processFacetStats: function(obj) {
+            // var divs = $("#amount div");
+            // var that = this;
+
+            // $("#slider-range-max").slider({
+            //     range: !0,
+            //         animate: !0,
+            //         min: obj[that.mappedFields.price].min,
+            //         max: obj[that.mappedFields.price].max,
+            //         values: [obj[[that.mappedFields.price]].values.min, obj[that.mappedFields.price].values.max],
+            //         create: function() {
+            //             jQuery("#amount").val("$" + obj[that.mappedFields.price].values.min + '  -  ' + "$" + obj[that.mappedFields.price].values.max);
+            //             divs.eq(0).html("$" + obj[that.mappedFields.price].values.min);
+            //             divs.eq(1).html("$" + obj[that.mappedFields.price].values.max);
+            //         },
+            //         slide: function(b, c) {
+            //             jQuery("#amount").val("$" + c.values[0] + '  -  ' + "$" + c.values[1]);
+            //             divs.eq(0).html("$" + c.values[0]);
+            //             divs.eq(1).html("$" + c.values[1]);
+            //         },
+            //         change: function(b, c) {
+            //             searchobj
+            //                 .clearRangeFiltes()
+            //                 .addRangeFilter(that.mappedFields.price, c.values[0], c.values[1])
+            //                 .setPage(1)
+            //                 .callResults(searchobj.paintResultSet, true);
+            //         }
+            // });
+
+            // $('input.unbxd-filter-by-price').on("click", function () {
+            //     searchobj
+            //         .clearRangeFiltes()
+            //         .addRangeFilter(this.mappedFields.price, slider_min, slider_max)
+            //         .setPage(1)
+            //         .callResults(searchobj.paintResultSet, true);
+            // })
+
+            // $('p.clear-unbxd-filter-by-price').on("click", function (e) {
+            //     event.preventDefault()
+
+            //     slider_min = window.unbxd_min;
+            //     slider_max = window.unbxd_max;
+
+            //     searchobj
+            //         .clearRangeFiltes()
+            //         .setPage(1)
+            //         .callResults(searchobj.paintResultSet, true);
+            // })
+        },
+        setDefaultFilters: function() {
+           
+        },
```

Comment:
is this empty function needed?

Reply:
This is part of default config, to show that this option is available. Just like how we have set fields to empty array to indicate that config existence.